app/models/notifier.rb

```diff
@@ -76,6 +77,9 @@ def message_notification(message)
       @replyurl = url_for(:host => SERVER_URL,
                           :controller => "message", :action => "reply",
                           :message_id => message.id)
+      @user_message_author = @from_user
```

Comment:
I don't think this variable really needs such a long name. I'd suggest something like `@author` would be fine and more concise.